test/fail_compilation/ctfe14465.d

```diff
@@ -1,22 +1,22 @@
-/*
-TEST_OUTPUT:
----
-fail_compilation/ctfe14465.d(19): Error: uncaught CTFE exception ctfe14465.E("message")
-fail_compilation/ctfe14465.d(22):        called from here: foo()
-fail_compilation/ctfe14465.d(22):        while evaluating: static assert(foo())
----
-*/
-class E : Exception
-{
-    this(string msg)
-    {
-        super(msg);
-    }
-}
-
-bool foo()
-{
-    throw new E("message");
-}
-
-static assert(foo());
+/*
+TEST_OUTPUT:
+---
+fail_compilation/ctfe14465.d(19): Error: uncaught CTFE exception ctfe14465.E("message")
+fail_compilation/ctfe14465.d(22):        called from here: foo()
+fail_compilation/ctfe14465.d(22):        while evaluating: `static assert(foo())`
+---
+*/
+class E : Exception
+{
+    this(string msg)
+    {
+        super(msg);
+    }
+}
+
+bool foo()
+{
+    throw new E("message");
+}
+
+static assert(foo());
```

Comment:
I didn't realize that change happened, but part of my submission process is to convert to LF line endings.